lib/getDeterministricAttributes.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-plusplus */
```

Comment:
can you please move these to the `specific line` you need muted?
- rule: avoid muting `es-lint` for an entire `file`
- instead: mute `individual` lines